packages/eslint-config/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@oakai/eslint-config",
```

Comment:
This makes a new package that matches the same naming convention as the prettier config